src/routes/page.scss

```diff
@@ -0,0 +1,57 @@
+@use "uswds-core" as * with (
+  $theme-show-notifications: false
+);
+
+.greeting-wrapper {
+  position: relative;
+  // Columns are 7 and 5 (approx 60% and 40%), so lighten opacity at 60%
+  background: linear-gradient(
+    90deg,
+    rgba($primary-darkest, 0.95) 0%,
+    rgba($primary-darkest, 0.9) 60%,
+    rgba($primary-darkest, 0.8) 100%
+  );
+  .greeting-background {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+  }
+  .greeting-content {
+    padding: 2rem;
+    color: $accent-warm;
+    .greeting-body {
+      :first-child {
+        margin-top: 0;
+      }
+      @include at-media("desktop") {
+        :last-child {
+          margin-bottom: 0;
+        }
+      }
+    }
+    .greeting-commissioner-portrait-wrapper {
+      display: flex;
+      align-items: center;
+      .greeting-commissioner-portrait {
+        border: 3px solid $accent-warm;
+        border-radius: 50%;
+        aspect-ratio: 1/1;
+        // center and reduce size on small phones
+        margin: 0 auto;
+        width: 125px;
+        @include at-media("mobile-lg") {
+          // place below byline on larger phones and tablets
+          margin: 0 0 0 auto;
+        }
+        @include at-media("desktop") {
+          // reset margin and width for desktop
+          margin: auto;
+          width: auto;
+          // Set min-width for blurhash to render at proper size
+          min-width: 350px;
```

Comment:
This might need to change depending on if we make updates to the `Image` component for the headshot. This is just a value that I thought looked good to ensure the image wasn't loading in as a small circle before blowing up to the actual size. Can probably be improved.